bingo.js

```diff
@@ -225,8 +244,7 @@ function prevColour(square)
 function anotherColour(square, increment)
 {
 	const colourSelection = COLOUR_SELECTIONS[COLOURCOUNT];
-	const currColour = getColourClass(square);
-	const currIndex = colourSelection.indexOf(currColour);
+	const currIndex = getColourClassIndex(square);
```

Comment:
This change introduces a bug. In "Green only" mode for colour count, a green square can't be turned back into a grey square. Getting index from `colourSelection`, rather than from `ALL_COLOUR` is important here.